include/zephyr/drivers/sensor.h

```diff
@@ -356,6 +356,10 @@ enum sensor_attribute {
 	SENSOR_ATTR_GAIN,
 	/* Configure the resolution of a sensor. */
 	SENSOR_ATTR_RESOLUTION,
+
+	/* Set the GBIAS for embedded sensor fusion in radians/s. */
+	SENSOR_ATTR_GBIAS,
```

Comment:
How is this different from setting `SENSOR_ATTR_OFFSET` for `SENSOR_CHAN_GYRO_XYZ`?

Reply:
I didn't notice/remember this attribute and it seems in fact suitable for this job. I see that is currently used for mag and accel offset, but it might be used for gyro as well. And it is just a number (no unit), so I guess I could give it rad/s in this contest. Thanks

Reply:
Yeah, attributes by themselves are unit less. That's one of the things https://github.com/zephyrproject-rtos/zephyr/pull/83077 attempts to address. Each attribute in the spec has no unit, but when an attribute instance is declared in the spec it needs to be bound to a unit, the goal is to allow:
* Attribute alone - for attributes that affect the whole device like power mode etc
* Attribute/channel - for attributes that affect a channel like offset
* Attribute/trigger - for attributes that affect a trigger like FIFO watermark (which doesn't really fit with a channel because multiple channels could fill the FIFO).